README.md

```diff
@@ -26,7 +26,7 @@ After you've read the preparation guide, refer to the deployment example for you
 
 ### Beta deployment
 
-These are beta versions of 1Password SCIM Bridge deployments and components. These deployments _should_ work, but aren't guaranteed and will change in the future.
+These are beta versions of 1Password SCIM Bridge deployments and components. These deployments _should_ work, but aren't guaranteed and will change in the future. See the [README](./beta/README.md) for more information about the "beta" designation.
```

Comment:
What if we put the title as 'New Deployment' and say something like

"We have checked that these deployments work without any issue, but since we haven't seen various use cases in different environments yet, it is not 100% guaranteed and may be improved in the future."

Reply:
"new deployments" may be my favourite so far.